git_externals/cli.py

```diff
@@ -190,7 +194,7 @@ def gitext_add(external, src, dst, branch, tag, ref):
 
     It requires one of --branch or --tag.
     """
-    from git_externals import load_gitexts, dump_gitexts
+    from git_externals import load_gitexts, dump_gitexts, normalize_gitexts, print_gitext_info
 
     git_externals = load_gitexts()
```

Comment:
this is not on you, but naming this variable the same as the `git_externals` is pretty confusing.